recipients_app/docu/api/openapi_specification_0.0.1.yaml

```diff
@@ -0,0 +1,391 @@
+    parameters:
+      - $ref: "#/components/parameters/RecipientId"
+      - $ref: "#/components/parameters/PaymentId"
+    post:
+      summary: Contest a payment
+      operationId: contestPayment
+      security:
+        - bearerAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required: [reason]
+              properties:
+                reason:
+                  type: string
+                  description: Reason for contesting the payment
+      responses:
+        "204":
+          description: Payment contested successfully
+
+  /recipients/{recipientId}/surveys:
+    parameters:
+      - $ref: "#/components/parameters/RecipientId"
+    get:
+      summary: List surveys assigned to a recipient
+      operationId: listSurveys
+      security:
+        - bearerAuth: []
+      responses:
+        "200":
+          description: Surveys found
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Survey"
+
+components:
+  parameters:
+    RecipientId:
+      in: path
+      name: recipientId
+      required: true
+      schema:
+        type: string
+        format: uuid
+        description: Unique identifier for the recipient
+
+    OrganizationId:
+      in: path
+      name: organizationId
+      required: true
+      schema:
+        type: string
+        format: uuid
+        description: Unique identifier for the organization
+
+    PaymentId:
+      in: path
+      name: paymentId
+      required: true
+      schema:
+        type: string
+        description: Unique identifier for the payment
+
+  schemas:
+    Phone:
+      type: object
+      properties:
+        phoneNumber:
+          type: integer
+          description: Phone number without country code
+          example: 971234567
+
+    Organization:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: Unique identifier for the organization
+        name:
+          type: string
+          description: Organization name
+        contactName:
+          type: string
+          description: Primary contact person name
+        contactNumber:
+          type: string
+          description: Primary contact phone number
+
+    Recipient:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          readOnly: true
+          description: Unique identifier for the recipient
+        firstName:
+          type: string
+          description: First name
+        lastName:
+          type: string
+          description: Last name
+        preferredName:
+          type: string
+          description: Preferred name
+        callingName:
+          type: string
+          description: Name used for calling
```

Comment:
here I'm not sure if it is not the same thing but legacy and current. To check with web / admin team